state/txn.go

```diff
@@ -492,9 +492,6 @@ func (txn *Txn) TouchAccount(addr types.Address) {
 	})
 }
 
-//nolint:godox
```

Comment:
Could we do something regarding panics? For example 2 out of the 3 panics are thrown in the `CleanDeleteObjects`. Couldn't we add error propagation there instead?

Reply:
Tried to fix it https://github.com/0xPolygon/polygon-edge/pull/1597/commits/81f4b5d709d3a736d47e29b6749c802e101b4123. If it turns out that it is not correct, we can revert it or re-work it.